components/web/apps/org.wso2.carbon.apimgt.admin.app/src/main/public/css/custom.css

```diff
@@ -321,4 +321,9 @@ a {
 
 .fw-number{
     color: black;
+}
+
+.checkbox{
```

Comment:
I think there is a formatting issue here. A space is missing before the bracket.